src/uv-common.c

```diff
@@ -513,7 +513,18 @@ int uv_fs_scandir_next(uv_fs_t* req, uv_dirent_t* ent) {
   uv__dirent_t** dents;
   uv__dirent_t* dent;
 
-  unsigned int* nbufs = uv__get_nbufs(req);
+  unsigned int* nbufs;
+  
+  /* Did we fail in the request? */
```

Comment:
Small nit, but instead of questions in these comments, could you phrase as something like "Check if the request failed."